### ISSUE TITLE
 Could not find a version that satisfies the requirement torch

### ISSUE THREAD

**Issue by joseakamaru** [allenai/allennlp#3232]
**Describe the bug**
I have followed the installation instructions mentioned on the Github Readme and from the Allenai Website, but I always received the following prompt on the command line. 

  ERROR: Could not find a version that satisfies the requirement torch<1.2,>=0.4.1 (from allennlp) (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2)
ERROR: No matching distribution found for torch<1.2,>=0.4.1 (from allennlp)

I have tried installing the conda environment, but that has failed also. I received the same error regardless of the method of installation.  

**To Reproduce**
Steps to reproduce the behavior
1. Install Anaconda and ensure that its part of your PATH
2. From the command prompt, run the following conda create -n allennlp python=3.6
2. Activate conda enviroment via 'conda activate allennlp' 
3. Then install the allennlp running the following on the command prompt: pip install allennlp
![image](https://user-images.githubusercontent.com/38084191/64574262-47ed3980-d334-11e9-9a26-e2cfeace579c.png)

**Expected behavior**
I expected a clean download and install of the allennlp.

**System (please complete the following information):**
 - OS: Windows 10 Pro
 - Python version: Python 3.6.9 and Python 3.7.3
 - AllenNLP version: trying to install from master
 - PyTorch version: (if you installed it yourself)

**Additional context**

**Comment by joelgrus** [comment]
technically allennlp is not supported on windows. I suspect that's a big part of your problem. I would try installing torch first separately. if that works, then you should be able to install allennlp afterward.

**Comment by joseakamaru** [comment]
I was able to determine the issue with the import Torch. I needed to
install the package Cuda on my device, which then enables the installation
of Torch. After installing Torch, I still have the same error prompt:

ERROR: Could not find a version that satisfies the requirement
torch<1.2,>=0.4.1 (from allennlp) (from versions: 0.1.2, 0.1.2.post1,
0.1.2.post2) ERROR: No matching distribution found for torch<1.2,>=0.4.1
(from allennlp)

Even though I have Torch (1.2.0) and torchvision (0.4.0) installed.

Jose Medina
The University of Alabama at Birmingham
Graduate School for Computer Engineering
( <%28813%29%20480-9750>205) 253 9350
jrmedina@uab.edu
